### ISSUE TITLE
[BITV] 9.1.4.11/6.2 - Note: Some social media links, implemented as clickable images (see screenshot), do not meet the contrast requirements for graphical controls. (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41860]
![image](https://github.com/nextcloud/server/assets/6078378/61983f8c-a7b1-4008-b43b-e6a380a59f24)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-3dbfb7615a-v6-n2

**Comment by szaimen** [comment]
I would say from design perspective lets just use black-white icons?
